build-profiling-ffi.sh

```diff
@@ -100,9 +103,14 @@ cp -v LICENSE LICENSE-3rdparty.yml NOTICE "$destdir/"
 export RUSTFLAGS="${RUSTFLAGS:- -C relocation-model=pic}"
 
 datadog_profiling_ffi="datadog-profiling-ffi"
-echo "Building the ${datadog_profiling_ffi} crate (may take some time)..."
+FEATURES="--features cbindgen"
+if [[ "$symbolizer" -eq 1 ]]; then
+    FEATURES="--features cbindgen,symbolizer"
+fi
+cargo build --features "cbindgen,symbolizer"
```

Comment:
Shouldn't this use `$FEATURES`

Reply:
this is a forgotten test. Thank you for spotting it :facepalm: 
build line is bellow